### ISSUE TITLE
[HL2] Stunsticks, SLAMs, and Episodic flashlights

### ISSUE THREAD

**Issue by Malavek** [ValveSoftware/Source-1-Games#6777]
Hi, since usable stunsticks were added in base Half-Life 2, I was wondering a few things:

First, so map creators don't need to turn on the command "stunstick_pickupable 1" at the start of their maps, I was wondering if the option could be added to the env_global entity for map creators to more easily enable it without messing with their players' consoles.
Second, will the SLAM be ported over as well given that it's fully coded and functional and just needs to be ported over the same way the stunstick was?
Third, for parity, will the Episode 2 flashlight be ported over to Half-Life 2? It wouldn't be the first time newer features were back-ported like with the updated vortigaunts and shotgun soldiers, plus it won't break any gameplay since the Ep2 flashlight is basically a "fixed" version of the 2004 one.
Finally, since a lot of bugs are still being fixed, is it still possible for fixing decal fading of the Tau Cannon and Stalkers to be on the table? I asked before but it was closed because someone else made a post about it, but there wasn't really a clear answer. Us modders (as far as I know) can't fix it for our mods since it's not in any released Half-Life 2 SDK code and is in the engine itself. It would greatly improve the base games but also allow modders to use the feature for our own weapon additions.

I really hope you'll consider this, thanks for reading.

**Comment by Marioiscool246** [comment]
With the stunstick, you can give yourself one even with stunstick_pickupable 0 by using the weapon_stunstick_player entity. The only difference with the other stunstick entity is that the particle effect will always be visible on the worldmodel.
redglowfade not fading properly was fixed too.

**Comment by Malavek** [comment]
> redglowfade not fading properly was fixed too.

wait fr?? I don't see it in the changelogs?

**Comment by Marioiscool246** [comment]
> > redglowfade not fading properly was fixed too.
> 
> wait fr?? I don't see it in the changelogs?

Yeah, I wanted to see if it was fixed out of curiosity when the 20th anniversary update released

**Comment by Malavek** [comment]
> > > redglowfade not fading properly was fixed too.
> > 
> > 
> > wait fr?? I don't see it in the changelogs?
> 
> Yeah, I wanted to see if it was fixed out of curiosity when the 20th anniversary update released

I checked when the initial update dropped and it did not appear to be fixed
